examples/get_started.ipynb

```diff
@@ -390,7 +390,7 @@
      "name": "stderr",
```

Comment:
<p>We should try to suppress this warning here too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/979/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>